.github/workflows/ci.yml

```diff
@@ -9,6 +9,13 @@ on:
   pull_request:
   workflow_dispatch:
 
+env:
+  APT_DEPS: >-
+    libglib2.0-dev
+    gcc
+    gettext
+    libpam0g-dev
```

Comment:
hum, are you sure this is needed for this project? The pam dep in particular sound like a copy from other projects where we need it :)